fe/src/main/java/org/apache/doris/qe/HeartbeatFlagHelper.java

```diff
@@ -0,0 +1,107 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.qe;
+
+import com.google.common.base.Joiner;
+import com.google.common.base.Splitter;
+import com.google.common.collect.Maps;
+import org.apache.doris.common.AnalysisException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class HeartbeatFlagHelper {
+
+    /* Bits for different Heartbeat Flag modes, you can add custom flags here */
+    public static final long FLAG_SET_DEFAULT_ROWSET_TYPE_TO_BETA = 1L;
```

Comment:
should define this flag in thrift to avoid define it multiple times.

Reply:
remove it.